web/wp-content/themes/dctx/header.php

```diff
@@ -11,6 +11,7 @@
 	<link rel="icon" type="image/png" href="<?php echo ROOT; ?>favicon.png">
 
 	<?php wp_head(); ?>
+	
```

Comment:
:warning: **Warning**: Found precision alignment of 1 spaces (*WordPress.WhiteSpace.PrecisionAlignment.Found*).